package.json

```diff
@@ -111,7 +112,7 @@
     "push-translations": "rm -rf src/.next/ && npx ttag update public/i18n/translations.pot ./src && tx push -s && echo \"Translators can now start translating new strings:\\n\\nhttps://www.transifex.com/sozialhelden/wheelmap-react-frontend/translate/#de/translationspot/335454735?q=translated%3Ano\"",
     "pull-translations": "tx pull -af && node scripts/parseGettextFiles.js",
     "retranslate": "npm run build && npm run pull-translations && NEW_SOURCE_PO='./src/public/i18n/en_US.txt' retranslate/run.sh",
-    "compile-svg-directory": "echo \"Compiling SVGs in ${basedir}, this can take up to a few minutes…\"; (for file in ${basedir}/*.svg; do node node_modules/@svgr/cli/bin/svgr --icon $file > ${file/.svg/.tsx}; sed -i '' -e 's/= props/= (props: React.SVGAttributes<SVGElement>)/g' ${file/.svg/.tsx}; echo \"export { default as `basename $file .svg` } from './`basename $file .svg`';\";  done;) > ${basedir}/index.ts",
+    "compile-svg-directory": "npx @svgr/cli --out-dir ${basedir} -- ${basedir}",
```

Comment:
This changes the svgr build prozess, as it now includes the functionality to create a index.ts file. This could introduce issues when recompiling other icons, I would like to tackle this in a separate PR.